lib/util/versioning.js

```diff
@@ -80,7 +80,14 @@ function get_runtime_abi(runtime, target_version) {
     if (runtime === 'node-webkit') {
         return get_node_webkit_abi(runtime, target_version || process.versions['node-webkit']);
     } else if (runtime === 'electron') {
-        return get_electron_abi(runtime, target_version || process.versions.electron);
+        var electron_version = target_version || process.versions.electron;
+        if (!electron_version) {
+            // TODO PR something to electron to pass in the version number for forks
+            // https://github.com/electron/electron/issues/9058
+            try { electron_version = require('electron/package.json').version; }
```

Comment:
If electron is installed as a global package, this will not work because `require` does not access global packages.